Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.4.1'
+ruby '2.5.1'
```

Comment:
why `2.5.1` and not `2.5.5`(highest patch version) or even `2.6.3` ?

Reply:
I would suggest updating to `2.5.5` in this PR, and we can create an issue to update to `2.6.3`. WDYT? @santib @GuilleLeopold